### PR TITLE
Huy pushing several scripts

### PR DESCRIPTION
**Pull request by HuyPham52**

### CODE REVIEW

File: src/mongodb_handling/data_process
Comment (NHNguyen94) [inline src/mongodb_handling/data_process:1]: Lack of file extension (.py)

Can we organize these function into a class and use @staticmethod

Also can you please add the db connection function into this class as you are doing it in the "main" function at the end of this file: client = MongoClient('localhost', 27017)

File: src/data_processing/json/json_normalize.py
Comment (NHNguyen94) [inline src/data_processing/json/json_normalize.py:1]: please add unit test for this

File: src/mongodb_handling/data_process
Comment (HuyPham52) [reply src/mongodb_handling/data_process:1]: sure

File: src/data_processing/json/json_normalize.py
Comment (HuyPham52) [reply src/data_processing/json/json_normalize.py:1]: Could I add the test scripts later? So in meantime please help me approve for the new changes @NHNguyen94

**Pull request merged**